html/changelogs/DNB.yml

```diff
@@ -0,0 +1,6 @@
+author: RyverStyx
+
+delete-after: True
+
+changes:
+  - tweak: "Adds important info to Nralakk Federation Passports for those abroad." 
```

Comment:
Very ambiguously worded for some reason?

Reply:
You're right, Addressed it